src/Argu/Utils.fs

```diff
@@ -152,7 +172,7 @@ type UnionCaseInfo with
 
         match uci.GetCustomAttributes typeof<'T> with
         | [||] when includeDeclaringTypeAttrs ->
-            match uci.DeclaringType.GetCustomAttributes(typeof<'T>, false) with
+            match uci.DeclaringType.GetCustomAttributes(typeof<'T>, false) |> Seq.toArray with
```

Comment:
On dotnetcore the `GetCustomAttributes` returns a sequence instead of an array. With this change no `#if` is required (and shouldn't be too bad as toArray should check if it's already an array internally).